beacon-chain/blockchain/forkchoice/process_attestation.go

```diff
@@ -147,6 +152,21 @@ func (s *Store) verifyAttPreState(ctx context.Context, c *ethpb.Checkpoint) (*pb
 	return baseState, nil
 }
 
+// verifyAttTargetEpoch validates attestation is from the current or previous epoch.
+func (s *Store) verifyAttTargetEpoch(ctx context.Context, genesisTime uint64, nowTime uint64, c *ethpb.Checkpoint) error {
+	currentSlot := (nowTime - genesisTime) / params.BeaconConfig().SecondsPerSlot
+	currentEpoch := helpers.SlotToEpoch(currentSlot)
+	var prevEpoch uint64
+	// Prevents previous epoch under flow
+	if currentEpoch > 1 {
```

Comment:
shouldnt this be `currentEpoch > 0` ?

Reply:
`prevEpoch` is 0 by default, the only time when `prevEpoch` can be 1 or more is when `currentEpoch` is 2 and more

I mostly followed the spec PR there, `currentEpoch > 0` works too